.tx/config

```diff
@@ -9,7 +9,7 @@ file_filter = MapboxNavigation/Resources/<lang>.lproj/Localizable.strings
 source_file = MapboxNavigation/Resources/Base.lproj/Localizable.strings
 source_lang = en
 
-[mapbox-navigation-core-ios.localizablestrings]
+[mapbox-navigation-ios.LocalizableCoreStrings]
 file_filter = MapboxCoreNavigation/Resources/<lang>.lproj/Localizable.strings
```

Comment:
Yeah, unfortunately, I think renaming any slug will lose any translations in locales that haven’t met the 80% threshold for pulling translations into this repository. But it’s better to do it now than later.